images/manageiq-base/Dockerfile

```diff
@@ -36,32 +35,31 @@ RUN chmod -R g+w /etc/pki/ca-trust && \
     chmod -R g+w /usr/share/pki/ca-trust-legacy
 
 # Install dumb-init to be used as the entrypoint
-RUN curl -L -o /usr/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_${ARCH} && \
+RUN ARCH=$(uname -m) && \
+    curl -L -o /usr/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_${ARCH} && \
     chmod +x /usr/bin/dumb-init
 
 COPY rpms/* /tmp/rpms/
 COPY container-assets/create_local_yum_repo.sh /
 COPY container-assets/clean_dnf_rpm /usr/local/bin/
 
-RUN curl -L https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo > /etc/yum.repos.d/ansible-runner.repo
+RUN curl -L https://releases.ansible.com/ansible-runner/ansible-runner.el9.repo > /etc/yum.repos.d/ansible-runner.repo
```

Comment:
This doesn't exist - this is part of the reason I'm moving runner to a python package instead.

Reply:
Fixed